modules/aws/sqs_cloudwatch.hcl

```
@@ -0,0 +1,133 @@
+ingester aws_sqs_cloudwatch module {
+  frequency  = 60
+  lookback   = 600
+  timeout    = 30
+  resolution = 60
+  lag        = 60
+
+  label {
+    type = "service"
+    name = "$input{service}"
+  }
+
+  physical_component {
+    type = "aws_sqs"
+    name = "$input{QueueName}"
+  }
+
+  data_for_graph_node {
+    type = "aws_sqs"
```

```suggestion
    type = "aws_sqs_queue"
```
This will not show up on UI because we are not sending data for any logical component, to solve this, we have to follow the same strategy we used for RDS. add `queue` suffix in `name` of `data_for_graph_node`.